zebra-consensus/src/error.rs

```diff
@@ -78,13 +78,17 @@ pub enum TransactionError {
     #[error("bindingSig MUST represent a valid signature under the transaction binding validating key bvk of SigHash")]
     RedJubjub(zebra_chain::primitives::redjubjub::Error),
 
+    #[error("bindingSig MUST represent a valid signature under the transaction binding validating key bvk of SigHash")]
+    RedPallas(zebra_chain::primitives::redpallas::Error),
```

Comment:
Should we distinguish between the curves in these error messages?
Or will the variant names always appear in the logs?

Reply:
Clarified anyway in 44e95cbf356c